modules/registrars/openprovider/Controllers/System/DomainSuggestionsController.php

```diff
@@ -72,9 +74,41 @@ public function get($params)
                 return mb_substr($tld, 1);
             }, explode(',', $suggestionSettings['suggestTlds']));
 
+        $placementLogin = Configuration::get('placementPlusAccount');
+        $placementPassword = Configuration::get('placementPlusPassword');
+        $isTestMode = $params['test_mode'] != 'on';
+        $usePlacementPlus = $placementLogin && $placementPassword && $isTestMode;
+
+
+        // Get placement domain suggestion
+        if ($usePlacementPlus) {
+            $encodedDomain = urlencode($params['searchTerm']);
+
+            $firstRankedDomain =
```

Comment:
move `$this->getSuggestedDomainFromPlacementPlus(` to the same line with `$firstRankedDomain =`